src/org/jitsi/impl/neomedia/rtp/TransportCCEngine.java

```diff
@@ -446,81 +468,19 @@ public void tccReceived(RTCPTCCPacket tccPacket)
     }
 
     /**
-     * Handles RTP packets for this {@link TransportCCEngine}.
+     * Gets the engine which handles outgoing RTP packets for this instance.
      */
-    private class RTPTransformer
-        extends SinglePacketTransformerAdapter
+    public TransformEngine getEgressEngine()
```

Comment:
just wondering, is there a reason this class can't still implement `TransformEngine` and return these where needed?  is it not possible to work this ingress/egress into the existing transformengine scheme we have everywhere else?